tpm2/tpm2.go

```diff
@@ -781,6 +781,12 @@ func encodePolicyPCR(session tpmutil.Handle, expectedDigest tpmutil.U16Bytes, se
 }
 
 // PolicyPCR sets PCR state binding for authorization on a session.
+//
+// Note that expectedDigest must be a *digest* of the expected PCR value. You
```

Comment:
We should note here that expected digest is optional. If you provide it, the digest is checked, if you don't provide it, the call will always succeed (as if you had passed the right digest).

Reply:
Done